fastlane/spec/spec_helper_spec.rb

```diff
@@ -23,16 +23,6 @@
       end
       expect(ARGV).to eq(current_ARGV)
     end
-
-    it "sets ARGV for good if no block is given" do
```

Comment:
Is this another feature that was dropped and not mentioned in the PR description?